doc/source/whatsnew/v0.25.2.rst

```diff
@@ -99,7 +99,7 @@ Sparse
 Other
 ^^^^^
 
--
+- Bug in :meth:`DataFrame.drop` when passing empty :meth:`DatetimeIndex` or :meth:`list`
```

Comment:
"meth" -> "class" for DatetimeIndex, not needed at all for list, which should instead have double ticks